core/chains/evm/logpoller/orm.go

```diff
@@ -240,59 +227,64 @@ type Exp struct {
 	ShouldDelete bool
 }
 
-func (o *DbORM) DeleteExpiredLogs(qopts ...pg.QOpt) error {
-	qopts = append(qopts, pg.WithLongQueryTimeout())
```

Comment:
@jmank88 I see occasionally we add this `WithLongQueryTimeout` context on top of the caller's ctx. Have we noticed time out issues in the past, or caller's context is enough?

Reply:
The caller's context is not the problem. This hook changes the underlying Q backing the ORM to use a longer timeout than it would apply by default. IMHO we need to preserve all of these options one way or another in order to minimize the impact of this change. I have a branch somewhere experimenting with some options but there isn't a very clean way to do it without assuming implementation details of the Q/Queryer.

There might be a way to shed the default timeout (at least in cases where there is an existing deadline), but it would have to be done very delicately.

Reply:
Going to resolve this discussion for now, I'd say we can create another PR later to tackle this change.